eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Scripts.hs

```diff
@@ -371,8 +384,9 @@ instance forall era. (Typeable (Crypto era), Typeable era) => ToCBOR (Script era
 
 encodeScript :: (Typeable (Crypto era)) => Script era -> Encode 'Open (Script era)
 encodeScript (TimelockScript i) = Sum TimelockScript 0 !> To i
-encodeScript (PlutusScript PlutusV1 s) = Sum (PlutusScript PlutusV1) 1 !> To s -- Use the ToCBOR instance of ShortByteString
-encodeScript (PlutusScript PlutusV2 s) = Sum (PlutusScript PlutusV1) 2 !> To s
+-- Use the ToCBOR instance of ShortByteString:
+encodeScript (PlutusScript PlutusV1 s) = Sum (PlutusScript PlutusV1) 1 !> To s
+encodeScript (PlutusScript PlutusV2 s) = Sum (PlutusScript PlutusV2) 2 !> To s
```

Comment:
I'd like to draw your attention to a bug that was fixed on this line. :wink: 

I didn't create a separate PR for this because as it turns out constructor supplied to `Sum` is only used for testing to check roundtripping.

Reply:
good catch, thanks!